lib/marked.js

```diff
@@ -4,6 +4,38 @@
  * https://github.com/markedjs/marked
  */
 
+// Return str with all trailing {c | all but c} removed
+// allButC: Default false
+function rtrim(str, c, allButC) {
```

Comment:
Should there be unit tests for this function?

Reply:
We would need to export the function. I think that would constitute [testing the implementation](https://testing.googleblog.com/2013/08/testing-on-toilet-test-behavior-not.html).

Reply:
I agree with @UziTech.